tests/pipeline/resources/parallel-pipeline/pipeline.yaml

```diff
@@ -0,0 +1,88 @@
+- type: producer-app
+  name: transaction-avro-producer-1
+  app:
+    replicaCount: 1
+    image: ${DOCKER_REGISTRY}/custom-transaction-producer
+    imageTag: 1.0.0
```

Comment:
do we really need to set this? it looks more like the atm-fraud example

Reply:
like can't you just remove the entire `app` section for all of these apps?

Reply:
like this?